application/templates/partials/_example.njk

```diff
@@ -11,35 +12,35 @@
   <div class="app-example-wrapper app-example__language-switch--en-activated" id="{{ exampleId }}" data-module="app-tabs app-language-switch-example">
     <div class="app-example app-example--tabs">
       <span class="app-example__link app-example__new-window">
-        <a href="{{ exampleURL }}" target="_blank">
+        <a href="{{ exampleURL }}" target="_blank" title="Open {{description}} example in a new window">
           Open this example in a new window
         </a>
       </span>
       {% if welshExampleURL %}
         <span class="app-example__link app-example__language-switch app-example__language-switch--current">
-          <a href="{{ exampleURL }}" data-lang="en" target="_blank" data-track="lang-switch-en">
+          <a href="{{ exampleURL }}" data-lang="en" target="_blank" data-track="lang-switch-en" title="Display {{description}} example in English">
             Display in English
           </a>
         </span>
         <span class="app-example__link app-example__language-switch js-enabled">
-          <a href="{{ welshExampleURL }}" data-lang="cy" target="_blank" data-track="lang-switch-cy">
+          <a href="{{ welshExampleURL }}" data-lang="cy" target="_blank" data-track="lang-switch-cy" title="Display {{description}} example in Welsh">
```

Comment:
Capitalisation will be a bit weird here:

`Display Badge with notifications example in Welsh"

I'm not convinced these sentences would provide much useful context.  I also don't have great advice on a better way.  My best guess would be:

`Display the welsh example for "Badge with notifications"`